waspc/src/Wasp/Generator/Job/IO.hs

```diff
@@ -29,6 +31,15 @@ readJobMessagesAndPrintThemPrefixed chan = runPrefixedWriter go
         J.JobOutput {} -> printJobMessagePrefixed jobMsg >> go
         J.JobExit {} -> return ()
 
+collectJobTextOutputUntilExitReceived :: Chan J.JobMessage -> IO [Text]
+collectJobTextOutputUntilExitReceived = collect []
```

Comment:
In haskell, usual idiom is to go with `go`, for functions like `collect`, as a name.